test/tz_test/validate.cpp

```diff
@@ -90,9 +90,9 @@ tzmain()
     auto& db = get_tzdb();
     std::vector<std::string> names;
     names.reserve(db.zones.size() + db.links.size());
-    for (auto& zone : db.zones)
+    for (const auto& zone : db.zones)
```

Comment:
Thanks for your comments. Why do you suggest that && is better? There is no intent to move or modify the elements being iterated so I would (and currently still do) think const auto& is better than &&. as it conveys that. This article:
http://stackoverflow.com/questions/15927033/what-is-the-correct-way-of-using-c11s-range-based-for
if I read it correctly seems to conclude the same opinion as me - note at the end comments about if "observing" prefer const auto&

If you think that article is wrong or I have read it incorrectly do point me to the correct reference. Thanks!